_posts/wip-reaching-out.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "2020-01-14: Interacting with the IAC Team"
+author: pmcmaw
+categories:
+  - team
+tags:
+  - community
+  - reviews
+  - modules
+  - windows
+  - communication
+---
+
+The IAC Team have decided to reach out to the community to make everyone aware of the best ways to get in contact with us. Whether this be to review a PR, highlight a nasty bug or really for any other reason. By doing this we hope to minimalise tasks falling through the cracks.
+
+## Monday is Community Day
+
+Yes, thats right! The whole team dedicates a full day to you every week. In this session we attempt to go over new issues, pull requests, releases and anything else module related.
+
+## Reaching out via Slack
+
+Weekly office hours on Monday 1.00pm and 5.00pm GTM alternating each week. Join #office-hours on our community-slack(puppetcommunity.slack.com). Between these hours the IAC team will be online and can chat about anything module related and much more!
+
+## Reaching out via Jira
+
+When you come across an issue with one of our supported modules and want to raise an issue, in order for to it to come directly to us there are a few things needed, this will ensure it lands on our teams plate and doesn't get lost in a large void of old and stale tickets.
+
+The values we would recommend setting are:
```

Comment:
Missing a note about
- [ ] please check that the issue hasn't been reported before by using the search function
- [ ] if the repository has issues enabled use that